.gitignore

```diff
@@ -5,6 +5,7 @@ build
 node_modules
 lib
 es
+/release/releaseSecret.js
```

Comment:
could you provide in the git repo a `/release/releaseSecret.js.sample` (with no secrets of course) so we can know the format?

Reply:
That's a very good point. I will

Reply:
+1 to this request